src/status_im/contexts/wallet/add_account/confirm_address_to_save/view.cljs

```diff
@@ -0,0 +1,103 @@
+(ns status-im.contexts.wallet.add-account.confirm-address-to-save.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.common.emoji-picker.utils :as emoji-picker.utils]
+    [status-im.common.not-implemented :as not-implemented]
+    [status-im.contexts.wallet.add-account.confirm-address-to-save.style :as style]
+    [status-im.contexts.wallet.common.screen-base.create-or-edit-account.view :as
+     create-or-edit-account]
+    [utils.debounce :as debounce]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def ^:const sheet-closing-delay 750)
+
+(defn- on-success-confirm-address
+  [theme]
+  (rf/dispatch [:navigate-back])
+  (debounce/debounce-and-dispatch
+   [:navigate-back]
+   sheet-closing-delay)
+  (debounce/debounce-and-dispatch
+   [:toasts/upsert
+    {:type  :positive
+     :theme theme
+     :text  (i18n/label
+             :t/address-saved)}]
+   sheet-closing-delay))
+
+(defn- on-press-confirm-address
+  [{:keys [account-name account-emoji account-color address ens? theme]}]
+  (rf/dispatch
+   [:wallet/save-address
+    {:address             address
+     :name                account-name
+     :customization-color account-color
+     :account-emoji       account-emoji
+     :ens                 (when ens? address)
+     :on-success          #(on-success-confirm-address theme)}]))
+
+(defn to-be-implemented
+  []
+  not-implemented/alert)
```

Comment:
I think you'll want to call this function `(not-implemented/alert)`, or replace `to-be-implemented` with `not-implemented/alert` for the `on-press` handler.

Reply:
Makes sense, It was still WIP and I haven't tested.
But thanks a ton for taking a closer look!